yarn.lock

```diff
@@ -640,6 +640,11 @@
   resolved "https://registry.yarnpkg.com/@types/node/-/node-13.13.15.tgz#fe1cc3aa465a3ea6858b793fd380b66c39919766"
   integrity sha512-kwbcs0jySLxzLsa2nWUAGOd/s21WU1jebrEdtzhsj1D4Yps1EOuyI1Qcu+FD56dL7NRNIJtDDjcqIG22NwkgLw==
 
+"@types/node@^14.0.5":
```

Comment:
Now we have two versions. Not sure why this happens though b/c `^14.0.5` also contains the above `14.11.2` version

Reply:
as I read it, we had two versions before, they are just listed differently now